src/Microsoft.DotNet.Metrics/Microsoft.DotNet.Metrics.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
+    <TargetLatestRuntimePatch>False</TargetLatestRuntimePatch>
+    <SignAssembly>false</SignAssembly>
+    <LangVersion>latest</LangVersion>
+    <RootNamespace>Microsoft.DotNet.Metrics</RootNamespace>
+    <AssemblyName>Microsoft.DotNet.Metrics</AssemblyName>
```

Comment:
These are both redundant. They are both inferred from the project name so these can be removed.

Reply:
Where are they inferred? I looked in the Dictionary.props/targets and didn't see anything. Or is this dark magic from the depths of dotnet core somewhere? I hate magic.

Reply:
In a .NET CLI project. RootNamespace and AssemblyName are automatically the project name.
This project should really be `dotnet new` + IsPackable and IsShipping properties